src/app/dataValidator/rules/topstories.js

```diff
@@ -0,0 +1,55 @@
+import deepClone from '../../helpers/json/deepClone';
+import deepGet from '../../helpers/json/deepGet';
+
+const squashKeys = [
+  'responsive-top-stories',
+  'top-story',
+  'secondary-top-story',
+  'other-top-stories',
+];
+
+const squashTopStories = jsonRaw => {
+  const json = deepClone(jsonRaw); // make a copy so we don't corrupt the input
+  let groups = deepGet(['content', 'groups'], json);
+  let collectedItems = [];
```

Comment:
```suggestion
  const collectedItems = [];
```

This doesn't need to be redeclared on [line 22](https://github.com/bbc/simorgh/pull/1596/files#diff-647ce4d4a42e13902b4fdc37c884a0bfR22) because we are just adding items to the array so can be a const. 

![image](https://user-images.githubusercontent.com/7791726/56302458-b76a4e80-6131-11e9-81be-7f8bf13c3ceb.png)

Reply:
![image](https://user-images.githubusercontent.com/11341355/56304264-ab808b80-6135-11e9-8903-3721b3ac59f8.png)
:O